### PR TITLE
CB-15299: freeipa e2etest for cleanup, CB-12300: freeipa e2etest for createBindUser

### PR DESCRIPTION
**Pull request by schfeca75**
* CB-15299: freeipa e2etest for cleanup
* CB-12300: freeipa e2etest for createBindUser

### CODE REVIEW

File: integration-test/src/main/java/com/sequenceiq/it/cloudbreak/testcase/e2e/freeipa/FreeIpaUpgradeTests.java [hunk 1]
```diff
@@ -173,4 +181,52 @@ private void addListDeleteDnsZonesBySubnet(com.sequenceiq.freeipa.api.client.Fre
...
+            String testFailPrefix) {
+        while (isRunning(operationStatus)) {
+            try {
+                Thread.sleep(OPERATION_CHECKING_PERIOD);
+            } catch (InterruptedException e) {
+                String errorMessage = testFailPrefix + " has been interrupted";
+                logger.error(errorMessage);
+                throw new TestFailException(errorMessage);
+            }
+            operationStatus = ipaClient.getOperationV1Endpoint().getOperationStatus(operationStatus.getOperationId(), accountId);
```
Comment (lacikaaa) [inline integration-test/src/main/java/com/sequenceiq/it/cloudbreak/testcase/e2e/freeipa/FreeIpaUpgradeTests.java:218]: please don't reassign values to method parameters

**Comment by schfeca75** [comment]
Retest this please

**Review by horadla23 — COMMENTED** [review]
from authz/API point of view, it LGTM

**Review by lacikaaa — CHANGES_REQUESTED** [review]

**Review by lacikaaa — APPROVED** [review]

**Review by doktoric — APPROVED** [review]

File: integration-test/src/main/java/com/sequenceiq/it/cloudbreak/testcase/e2e/freeipa/FreeIpaUpgradeTests.java [hunk 2]
```diff
@@ -173,4 +181,42 @@ private void addListDeleteDnsZonesBySubnet(com.sequenceiq.freeipa.api.client.Fre
             throw new TestFailException("DNS ZONE test failed during upgrade with: " + e.getMessage(), e);
         }
     }
+
+    private void createBindUser(TestContext testContext, com.sequenceiq.freeipa.api.client.FreeIpaClient ipaClient, String environmentCrn, String accountId) {
+        BindUserCreateRequest bindUserCreateRequest = new BindUserCreateRequest();
```
Comment (doktoric) [inline integration-test/src/main/java/com/sequenceiq/it/cloudbreak/testcase/e2e/freeipa/FreeIpaUpgradeTests.java:186]: any loG ?

**Pull request merged**